tests/Conference/ConferenceHarvesterTest.php

```diff
@@ -0,0 +1,136 @@
+    }
+
+    public function provideFetcherResponses()
+    {
+        $testConference = new Conference();
+        $testConference
+            ->setName('TestConference')
+            ->setStartAt(new \DateTime());
+
+        $differentTestConference = new Conference();
+        $differentTestConference
+            ->setName('OtherTestConference')
+            ->setStartAt(new \DateTime('+1 year'));
+
+        $ignoredTestConference = new Conference();
+        $ignoredTestConference
+            ->setName('Java Conference');
+
+        yield 'test fetchers add new conference' => [
+            [
+                'name' => TululaFetcher::class,
+                'isActive' => true,
+                'conference' => $testConference,
+                'existingConference' => null,
+                'results' => ['updated' => 0, 'added' => 1],
+            ],
+        ];
+
+        yield 'test fetchers update existing conference' => [
+            [
+                'name' => TululaFetcher::class,
+                'isActive' => true,
+                'conference' => $testConference,
+                'existingConference' => $differentTestConference,
+                'results' => ['updated' => 1, 'added' => 0],
+            ],
+        ];
+
+        yield 'test inactive fetcher' => [
+            [
+                'name' => TululaFetcher::class,
+                'isActive' => false,
+                'conference' => $testConference,
+                'existingConference' => null,
+                'results' => ['updated' => 0, 'added' => 0],
+            ],
+        ];
+
+        yield 'conference should be ignored' => [
+            [
+                'name' => TululaFetcher::class,
+                'isActive' => true,
+                'conference' => $ignoredTestConference,
+                'existingConference' => null,
+                'results' => ['updated' => 0, 'added' => 0],
+            ],
+        ];
+    }
+
+    private function createHarvester(array $fetcherData)
+    {
+        $fetcherProphecy = $this->prophesize($fetcherData['name']);
+        $fetcherProphecy
+            ->fetch([])
+            ->willYield([$fetcherData['conference']]);
+
+        $fetcherConfigurationRepository = $this->prophesize(FetcherConfigurationRepository::class);
+        $fetcherConfigurationRepository
+            ->findOneOrCreate(Argument::type('string'))
+            ->willReturn($fetcherConfiguration = new FetcherConfiguration('Test'));
+        $fetcherConfiguration->setActive($fetcherData['isActive']);
+
+        $conferenceFilterRepository = $this->prophesize(ConferenceFilterRepository::class);
+        $conferenceFilterRepository
+            ->findAll()
+            ->willReturn([$filter = new ConferenceFilter()]);
+        $filter->setName('java');
+
+        $conferenceRepository = $this->prophesize(ConferenceRepository::class);
+        $conferenceRepository
+            ->findExistingConference(Argument::type('App\Entity\Conference'))
```

Comment:
```suggestion
            ->findExistingConference(Argument::type(Conference::class))
```

it's more natural :)